build.gradle.kts

```diff
@@ -0,0 +1,139 @@
+plugins {
+    alias(libs.plugins.kotlin.jvm)
+    id("java-gradle-plugin")
+    id("maven-publish")
+    alias(libs.plugins.nexus.publishing)
+    id("signing")
+}
+
+group = "io.cloudflight.gradle"
+if (System.getenv("RELEASE") == null) {
```

Comment:
I think we should also check if `RELEASE=true`. Otherwise `RELEASE=false` would also result in a final version